app/assets/stylesheets/blog.css

```
@@ -0,0 +1,1677 @@
+  margin-bottom: var(--space-8);
+  font-size: var(--font-size-lg);
+}
+
+.blog-post-body blockquote {
+  border-left: 4px solid #14b8a6;
+  padding-left: var(--space-20);
+  margin: var(--space-24) 0;
+  font-style: italic;
+  color: var(--color-text-secondary);
+  background: var(--color-surface-secondary);
+  padding: var(--space-20);
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)